app/decorators/event_decorator.rb

```diff
@@ -8,6 +8,10 @@ def event_type
   end
 
   def happened_at
-    EasternTimePresenter.new(event.created_at).to_s
+    event.created_at
```

Comment:
Changing this method to return a datetime value for comparison purposes, otherwise the string value was being used for comparisons.